eml/components/courses/courseCard/CourseCard.jsx

```diff
@@ -31,7 +31,7 @@ export default function CourseCard({ course, isOnline }) {
 
   const layout = downloaded || isOnline ? enabledUI : disabledUI;
 
-  let isDisabled = layout === disabledUI ? true : false;
+  let isDisabled = layout === disabledUI;
```

Comment:
why change?

Reply:
The last part war redundant. The statement said if true then true, if false then false, so now it just take the boolean from the equality and don't do the whole trinary statement that returned the same thing